### PR TITLE
Add Avali

### PR DESCRIPTION
**Pull request by Vermidia**
## About the PR

Ported over from https://github.com/cosmatic-drift-14/cosmatic-drift/pull/189
It was already given the green light, I just helped porting it.

## Why / Balance

More species, more customizability.

## Technical details

Xenos call them Four Eared Hosts, since it's different enough that they won't get confused with Vox (I will port...one day...I still have hope) and unique enough that it's unlikely to be confused with something else.

## Media

base
custom

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added Avali species

### CODE REVIEW

File: Resources/Audio/_RMC14/Voice/Avali/attributions.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- files: ["avali_scream"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Taken from Steam mod / Starbound. Made by Steam user: https://steamcommunity.com/id/wonky2 with permission."
+  source: "https://steamcommunity.com/sharedfiles/filedetails/?id=3001955034"
+
+- files: ["avali_1, avali_1_ask, avali_1_exclaim, avali_2, avali_2_ask, avali_2_exclaim"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Taken from Steam mod / Starbound. Made by Steam user: https://steamcommunity.com/id/Nefuki with permission."
+  source: "https://steamcommunity.com/sharedfiles/filedetails/?id=3164757879"
```
Comment (DrSmugleaf) [inline Resources/Audio/_RMC14/Voice/Avali/attributions.yml:9]: Are these two NC? It's unclear on whether or not having a Patreon conflicts with the CC NC licenses so we might have to get written permission from the author

Comment (Vermidia) [reply Resources/Audio/_RMC14/Voice/Avali/attributions.yml:9]: I see, I'll check

**Comment by Vermidia** [comment]
image
Got permission for the scream sound. Might take a day or 2 to check if the talk sounds are ok.

**Comment by Vermidia** [comment]
Was decided that this PR will drop the speech sounds for now, in lieu of new ones and/or permission.

**Review by QuietlyWhisper — COMMENTED** [review]
Can you post pictures with the avali in armor and bursted?

Can you also make sure all sounds and emotes are working just to be sure?

**Comment by Vermidia** [comment]
armor
tucked

https://github.com/user-attachments/assets/d3b3c8f4-18f9-42b0-b8ec-483489f582ab

(Them in armor, tucked helmet, and their currently only custom sound for now, a scream)

**Comment by Vermidia** [comment]
burst
Forgot bursted

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**